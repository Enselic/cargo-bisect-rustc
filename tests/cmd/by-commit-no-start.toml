bin.name = "cargo-bisect-rustc"
args = "--end 2024-03-01 --by-commit"
status = "failed"
