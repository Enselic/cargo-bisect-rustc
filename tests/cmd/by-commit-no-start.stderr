ERROR: --by-commit with an end date requires --start to be specified
